Lib/logging/handlers.py

```diff
@@ -812,6 +812,7 @@ def __init__(self, address=('localhost', SYSLOG_UDP_PORT),
         self.address = address
         self.facility = facility
         self.socktype = socktype
+        self.formatter = None
```

Comment:
This should not be needed, as it is done in the base class `__init__()`, which is invoked on line 810.